docs/python_manual.rst

```diff
@@ -0,0 +1,26 @@
+=================================
+``rasterio`` Python User's Manual
+=================================
+
+This user's manual is for Python developers who want a clean API for accessing raster data.
+
+.. todo::
+
+    What does it do, narrative examples
+
+    What it does NOT do
+
+    For command line tools look to CLI users manual 
+
+.. toctree::
+    :maxdepth: 2
+
+    installation
+    data_model
+    reading
+    working_with_datasets
+    writing
+    topics
+    osgeo_gdal_migration
```

Comment:
@perrygeo this seems like it should go to the end of the list (here and in the main TOC)